src/super_gradients/training/utils/callbacks.py

```diff
@@ -342,16 +370,16 @@ def __init__(self, max_epochs, **kwargs):
         super(PolyLRCallback, self).__init__(Phase.TRAIN_BATCH_STEP, **kwargs)
         self.max_epochs = max_epochs
 
-    def __call__(self, context: PhaseContext):
-        # POLYNOMIAL LEARNING RATE
-        if self.training_params.lr_warmup_epochs <= context.epoch:
-            effective_epoch = context.epoch - self.training_params.lr_warmup_epochs
-            effective_max_epochs = self.max_epochs - self.training_params.lr_warmup_epochs
+    def perform_scheduling(self, context):
+        effective_epoch = context.epoch - self.training_params.lr_warmup_epochs - self.training_params.lr_cooldown_epochs
```

Comment:
why do you reduce lr_cooldown_epochs here?

Reply:
fixed